### ISSUE TITLE
Privacy policy

### ISSUE THREAD

**Issue by alanjosephwilliams** [codeforamerica/clean#243]

**Comment by daguar** [comment]
@RebeccaCoelius — do you think this is something you could help us out with? I feel like you have the best connections to the world where we can find a reasonable one.

**Comment by daguar** [comment]
A few resources:
- http://www.iubenda.com/en
- http://www.startupcommons.org/privacy-policy.html

**Comment by RebeccaCoelius** [comment]
Just sent a message to Dharmishta as it might be a thing provided to
accelerator companies. If not, I'm going to ask Chas, then our lawyer
Deven. Should be no problem.

Rebecca Coelius, MD
Code for America
415 298 2872
LinkedIn <http://www.linkedin.com/in/rebeccacoelius>
Twitter <http://twitter.com/#!/RebeccaCoelius>

**Comment by RebeccaCoelius** [comment]
Dharmishta just posted a TOS from one of the companies in the #companies
slack channel, lmk if that works or you would like to cast a wider net

Rebecca Coelius, MD
Code for America
415 298 2872
LinkedIn <http://www.linkedin.com/in/rebeccacoelius>
Twitter <http://twitter.com/#!/RebeccaCoelius>

**Comment by RebeccaCoelius** [comment]
@daguar did Chas help you Thursday or do I need to follow up?

**Comment by daguar** [comment]
Short version of the advice from the sample of start-up people I've talked to is just to look at the policies for a few companies you like and take one (keeping in mind copyright, yadayada).

One approach I suggested that someone said sounded great was to look at ~3, find the common bits we like, and make it ours.

**Comment by daguar** [comment]
Assigned @lippytak per IRL convo.

**Comment by RebeccaCoelius** [comment]
Did you get Chas'?

Rebecca Coelius, MD
Code for America
415 298 2872
LinkedIn <http://www.linkedin.com/in/rebeccacoelius>
Twitter <http://twitter.com/#!/RebeccaCoelius>

**Comment by lippytak** [comment]
@cydharrell do we have a boilerplate privacy policy/terms of use for CfA applications that we could modify for Clean? Or maybe @dharmishta you have something from the Accelerator? Thanks!

**Comment by lippytak** [comment]
Will look into:
- tumblr
- amazon

**Comment by RebeccaCoelius** [comment]
we already asked @dharmistha

Rebecca Coelius, MD
Director of Health
Code for America
415 298 2872
LinkedIn <http://www.linkedin.com/in/rebeccacoelius>
Twitter <http://twitter.com/#!/RebeccaCoelius>

**Comment by RebeccaCoelius** [comment]
This is silly, Dave or Jake just email Deven Cc'ing me and let's create our
CfA optimal boilerplate. Then we can share w/ Fellows etc.

On Monday, February 2, 2015, Rebecca Coelius <rlmitchell1@gmail.com> wrote:

> we already asked @dharmistha
>
> Rebecca Coelius, MD
> Director of Health
> Code for America
> 415 298 2872
> LinkedIn <http://www.linkedin.com/in/rebeccacoelius>
> Twitter <http://twitter.com/#!/RebeccaCoelius>
>
> On Mon, Feb 2, 2015 at 3:53 PM, Jake Solomon <notifications@github.com
> <javascript:_e(%7B%7D,'cvml','notifications@github.com');>> wrote:
>
>> Will look into:
>>
>>    - tumblr
>>    - amazon
>>
>> —
>> Reply to this email directly or view it on GitHub
>> <https://github.com/codeforamerica/clean/issues/243#issuecomment-72535411>
>> .
>>
>
>

-- 
Rebecca Coelius, MD
Director of Health
Code for America
415 298 2872
LinkedIn <http://www.linkedin.com/in/rebeccacoelius>
Twitter <http://twitter.com/#!/RebeccaCoelius>

**Comment by daguar** [comment]
Yeah, that's actually a really great idea.

I think it'd be great to get out of this (1) a general CfA-reusable privacy
policy, and (2) a more specific health/sensitive data one (if necessary,
not sure).


On Mon, Feb 2, 2015 at 1:03 PM, RebeccaCoelius <notifications@github.com>
wrote:

> This is silly, Dave or Jake just email Deven Cc'ing me and let's create
> our
> CfA optimal boilerplate. Then we can share w/ Fellows etc.
>
> On Monday, February 2, 2015, Rebecca Coelius <rlmitchell1@gmail.com>
> wrote:
>
> > we already asked @dharmistha
> >
> > Rebecca Coelius, MD
> > Director of Health
> > Code for America
> > 415 298 2872
> > LinkedIn <http://www.linkedin.com/in/rebeccacoelius>
> > Twitter <http://twitter.com/#!/RebeccaCoelius>
> >
> > On Mon, Feb 2, 2015 at 3:53 PM, Jake Solomon <notifications@github.com
> > <javascript:_e(%7B%7D,'cvml','notifications@github.com');>> wrote:
> >
> >> Will look into:
> >>
> >> - tumblr
> >> - amazon
> >>
> >> —
> >> Reply to this email directly or view it on GitHub
> >> <
> https://github.com/codeforamerica/clean/issues/243#issuecomment-72535411>
> >> .
> >>
> >
> >
>
> --
> Rebecca Coelius, MD
> Director of Health
> Code for America
> 415 298 2872
> LinkedIn <http://www.linkedin.com/in/rebeccacoelius>
> Twitter <http://twitter.com/#!/RebeccaCoelius>
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/codeforamerica/clean/issues/243#issuecomment-72537201>
> .
>



-- 
Dave Guarino
Engineer, Health
(2013 Fellowship alumnus)
Code for America <http://www.codeforamerica.org/>
dave@codeforamerica.org
LinkedIn <http://www.linkedin.com/in/daveguarino/> | GitHub
<https://github.com/daguar/>

**Comment by lippytak** [comment]
Sent + asking @RebeccaCoelius to flesh out cross-project needs. Deven's out most of this week so I may move this to next sprint if we don't get a response soon.

**Comment by lippytak** [comment]
Deven said she would have us a draft early next week.

**Comment by lippytak** [comment]
First draft: https://github.com/codeforamerica/clean/wiki/Privacy-Policy

Will chat with Deven on the phone later this week.

**Comment by daguar** [comment]
We'll need to revise semi-significantly the paragraph starting with "CleanAssist stores your [identifiable] CalFresh Information only briefly..." since we've moved to storing case information for follow up.

As a more general comment — a lot of this goes into specifics of implementation (e.g., encryption, temporary storage) that I think are likely to change over time. I'd suggest that we put together a broader / more flexible policy as a result.

My understanding the basic need being met here is that we _have_ a policy that is clear, rather than specifically delineating everything we do.

Ideally, I'd like us to have a policy that provides the flexibility to do things like:

- Store case information for the life cycle of the application, to be able to follow up on clients' behalf (what we're already doing)
- Potentially provide case information back to the application assistors in the community that were helping a specific person apply, as they may be the best touch point
- Use Google Analytics
- Share aggregate information on clients based on case info (eg, avg # of household members)

The TLDR to me is, "we don't share your individual personal info with third parties except the agency (and state), your assistor if you chose that."

Other nitpick — we should not just say DSS, we are providing to the county and we should say explicitly that we provide it to CDSS and/or the county social services agency responsible for CalFresh enrollement.

**Comment by lippytak** [comment]
@daguar :+1:

Deven had some specific questions she wanted to review before taking another pass and I'll raise some of this too. But then I'll plan to make my changes and bring to you and @alanjosephwilliams + @RebeccaCoelius for a gut check - will probably want a full review then.

**Comment by lippytak** [comment]
Guide for google analytics privacy policy: http://www.iubenda.com/blog/2013/09/23/privacy-policy-google-analytics/

**Comment by lippytak** [comment]
@daguar Could you confirm:
- Currently we're only using cookies to store session ID (data remains server-side)
- In the near future we'll also be using cookies to track analytics data (via Google Analytics)

Any other uses of cookies I'm not aware of?

Also FYI let's not promise at-rest encryption in the policy right now as we would have to upgrade to the next tier (https://www.heroku.com/pricing#postgres).

**Comment by lippytak** [comment]
bump @daguar

**Comment by daguar** [comment]
Reassigned to me so it's on my waffle since it's in my court

**Comment by lippytak** [comment]
Draft below based on:
- http://www.coveredca.com/privacy/
- https://automattic.com/privacy/
- http://getremix.com/privacy (YC defaultish)
- https://www.healthcare.gov/privacy/

Things to consider later:
- Data retention - either "We currently retain all data indefinitely" OR "We only store your personal information for as long as we are using it to help you get CalFresh."
- All staff should agree not to share information during onboarding so we can say so in this policy (cc @jennymt @cydharrell )
- How to deal with third party services (Twilio, Front, Textitin, Hellofax, Getdrip (moving off), Intercom)? I literally couldn't find a single example that included this stuff.
- Will eventually need to add something about how we implement Do Not Track (CA req)
- Child stuff (<13 not allowed or something?)

===

# Privacy Policy
GetCalFresh.org is a non-profit website run by Code for America Labs, Inc. ("Code for America", "we", "us", "our"). This Privacy Policy describes how the GetCalFresh.org website (the "Site") collects, uses, and protects your personal information. By submitting your personal information on the Site, you agree to the terms in this Privacy Policy. If you do not agree with these terms, please do not use the Site.

# Overview
- We will not collect personal information from you without your knowledge and consent.
- We will only use your personal information to help you get and keep CalFresh.
- We will not knowingly disclose or sell your personal information to any third party, except as provided in this privacy policy.
- We will take all reasonable measures to protect your personal information.

# The personal information we collect
We collect all information that you provide within your application for CalFresh, including your name, address, contact information, social security number, financial information, and similar information for other individuals that you include on your CalFresh application. We also collect and store copies of any CalFresh documents you upload, like pictures of your Drivers License, paystubs, birth certifications, rent receipts, etc. We do not store any of this information unless you electronically sign your CalFresh application and click 'Submit Application' at the end.

In addition to the information you provide, we automatically collect and store information about your visit to the Site:
- Domain from which you access the Internet
- IP address (an IP or internet protocol address is a number that is automatically given to a computer connected to the Web)
- Operating system on your computer and information about the browser you used when visiting the site
- Date and time of your visit
- Pages you visited
- Address of the website that connected you to the Site (such as google.com or bing.com)

# How we use your information
We only use your personal information to help you get and keep CalFresh:
- We submit your CalFresh application to the appropriate department in your County.
- We send you a confirmation message of your CalFresh application, depending on your contact preferences.
- We remind you to complete your CalFresh phone interview, depending on your contact preferences.
- We remind you to submit CalFresh documents, depending on your contact preferences.
- We may provide ongoing support to help you keep CalFresh by reminding you when you have to submit income changes, complete your Semi-Annual Report, or complete your annual Recertification.

We will not email you unless you consent to email communication in the application and every email will include a link to unsubscribe.

We will not send you a text message unless you consent to text message communication in the application and you can reply 'STOP' to any text message to unsubscribe.

# How we share your information
Once you submit your CalFresh application on the Site, we share this information with the Human Services Agency (HSA) in the County where you live. Each California County has their own HSA, which processes CalFresh applications and administers CalFresh benefits. Once your personal information is received by the HSA it is then subject to their respective privacy policies. All California HSAs are listed here: www.cwda.org/links/chsa.php

We also disclose your personal information to Code for America employees, contractors, and affiliated organizations that need to know the information in order to provide the services of the Site.

Besides County HSAs and individuals affiliated with the Site, we do not share your personal information with any third parties, except when and to the minimal extent required by law.

# How we protect your information
Protecting your personal information is extremeley important to us and we take all reasonable measures to do so. Here are some specific security practices we use:
- When you enter your information on our site we encrypt the transmission of that information using transport layer security (TLS).
- We store your personal information on AWS GovCloud, which is a trusted third party data storage service designed specifically for US government agencies with sensitive information. For more information on see AWS GovCloud security, see: aws.amazon.com/govcloud-us/security/
- We only store your personal information for as long as we are using it to help you get CalFresh.

Although we try our best to protect the privacy of your personal information, we cannot guarantee complete security. The transmission of information via the Internet is not completely secure. Unauthorized entry or use, hardware or software failure, and other factors, may compromise the security of user information at any time.

# Changes to this policy
While most changes will be minor, we may change this Privacy Policy from time to time. Please check this page frequently for updates as your continued use of this site after any changes in this Privacy Policy will constitute your acceptance of the changes.

# Question
If you have any questions, comments, concerns, or complaints with the Site, please contact us by email at hello@getcalfresh.org, by phone at (415) 625-9633, or by mail at:

GetCalFresh
155 9th St
San Francisco, CA 94103

We will do our best to resolve the issue.

**Comment by lippytak** [comment]
@ajw @daguar @jennymt please review ^

Assuming we move to GovCloud by EOD Mon then I'd be comfortable putting this up with one consideration: Either we have a convo about data deletion and retain "We only store your personal information for as long as we are using it to help you get CalFresh" OR we don't have that convo and change this to "We currently retain all data indefinitely"

**Comment by lippytak** [comment]
Sorry wrong repo...more emails coming!

**Comment by lippytak** [comment]
https://github.com/codeforamerica/getcalfresh.org/issues/145
